code/game/objects/items/weapons/storage/fancy.dm

```diff
@@ -226,17 +256,24 @@
 	icon = 'icons/obj/cigarettes.dmi'
 	icon_state = "cigarcase"
 	w_class = 3
-	storage_slots = 7
+	storage_slots = 5
 	can_hold = list(/obj/item/clothing/mask/cigarette/cigar)
 	icon_type = "premium cigar"
 	spawn_type = /obj/item/clothing/mask/cigarette/cigar
 
+/obj/item/weapon/storage/fancy/cigarettes/cigars/CtrlClick(mob/user)
+	var/obj/item/weapon/storage/fancy/F = src
+	return F.CtrlClick(user)

```

Comment:
I... what? I'm not sure what you're trying to do here, but this shouldn't do anything.